charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -1,21 +1,169 @@
+            clientRoles:
+              account:
+                - "manage-account"
+                - "view-profile"
+          - username: modeler
+            enabled: true
+            firstName: "modeler"
+            lastName: User
+            email: "modeler@test.com"
+            credentials:
+            - type: password
+              value: password
+            realmRoles:
+              - "offline_access"
+              - "uma_authorization"
+              - "ACTIVITI_MODELER"
+            clientRoles:
+              account:
+                - "manage-account"
+                - "view-profile"
+          - username: hruser
+            enabled: true
+            firstName: "hruser"
+            lastName: hruser
+            email: "hruser@test.com"
+            credentials:
+            - type: password
+              value: password
+            realmRoles:
+              - "offline_access"
+              - "uma_authorization"
+              - "ACTIVITI_USER"
+            clientRoles:
+              account:
+                - "manage-account"
+                - "view-profile"
+            groups:
+              - "hr"
+          - username: hradmin
+            enabled: true
+            firstName: "hradmin"
+            lastName: hradmin
+            email: "hradmin@test.com"
+            credentials:
+            - type: password
+              value: password
+            realmRoles:
+              - "offline_access"
+              - "uma_authorization"
+              - "ACTIVITI_USER"
+              - "ACTIVITI_ADMIN"
+            clientRoles:
+              account:
+                - "manage-account"
+                - "view-profile"
+            groups:
+              - "hr"
+          - username: testuser
+            enabled: true
+            firstName: "testuser"
+            lastName: testuser
+            email: "testuser@test.com"
+            credentials:
+            - type: password
+              value: password
+            realmRoles:
+              - "offline_access"
+              - "uma_authorization"
+              - "ACTIVITI_USER"
+            clientRoles:
+              account:
+                - "manage-account"
+                - "view-profile"
+            groups:
+              - "testgroup"
+          - username: testadmin
+            enabled: true
+            firstName: "testadmin"
+            lastName: testadmin
+            email: "testadmin@test.com"
+            credentials:
+            - type: password
+              value: password
+            realmRoles:
+              - "offline_access"
+              - "uma_authorization"
+              - "ACTIVITI_USER"
+              - "ACTIVITI_ADMIN"
+            clientRoles:
+              account:
+                - "manage-account"
+                - "view-profile"
+            groups:
+              - "testgroup"
+        client:
+          webOrigins:
+            - "*"
+          redirectUris:
+            - "*"
+  # nginx-ingress:
```

Comment:
Is this commented-out paragraph really necessary?

Reply:
nope, removed.